js/Wikifier.js

```diff
@@ -2,23 +2,21 @@
 //-- Wikifier
 //--
 
-function getParser(tiddler,format)
+//# "parser" = "custom or default formatter" (should be just 1 term actually)
+function getParser(tiddler, format)
 {
 	if(tiddler) {
-		if(!format)
-			format = tiddler.fields["wikiformat"];
+		if(!format) format = tiddler.fields["wikiformat"];
 		var i;
+		//# format field takes precedence over format tag
 		if(format) {
-			//# format field takes precedence over format tag
-			for(i in config.parsers) {
```

Comment:
This is a good point, should probably revert this bit; but this is also something to take into account by the linter. Do you think it's good enough to only allow

    if(...) expression;

and

    if(...) {
        expression;
    }

and disallow

    if(...)
        expression;

?

(and thanks for the review)

Reply:
I personally would always go with option 2. 
From my professional programmers live, there have been way too many problems with the other 2.

Problems caused by changes from others or by myself create issues that could cost hours or even days to debug. ... I'm very conservative about this one. It did "bite" me and others quite a bit ;)

Reply:
I see. I think I'll start with banning option 3; there's no watching that enforces linting in the repo though, ideally it should be introduced as well.

Reply:
Pmario is right, forget no 3 and give preference to no 2, no 1 only if it clear from the code context what is done.

Reply:
I've done a small research (whether to use [curly](https://eslint.org/docs/latest/rules/curly) and what variant) with testing and found that in fact it's not that necessary. You see, the problem about

```
if(...)
    expression;
```

is that if it is allowed, it's easy to run into one of these pitfalls:

```
if(...);
    expression;

if(...)
    expression;
    expression2;
```

But in reality, both are already banned by linter complaining about wrong indentation. I've found that I haven't committed the [`indent` rule](https://eslint.org/docs/latest/rules/indent) yet (wanted to do so before I finish updating the whole core), but pushed now so that you can test and see.